apps/coscrad-frontend/src/app/App.spec.tsx

```diff
@@ -12,14 +12,18 @@ export const getDummyConfigurableContent = (): ConfigurableContent => ({
 
```

Comment:
I know this is my fault- but we should break out `getDummyConfigurableContent` to the`test-utils` directory.

Reply:
sounds good